tests/unit/test_handle_overrides.py

```diff
@@ -93,10 +94,12 @@ def test_handle_overrides_handles_imports_with_issues(input_txt):
         issues = []
         overrides = handle_overrides(input_txt, issues)
     assert overrides == {}
-    if PY3:
+    if PY2:
```

Comment:
I think we can presume the PY2 stuff can be deleted.

Reply:
Cool - #16 should be easy pickings